### ISSUE TITLE
Time in event list seems not to use correct timezone 

### ISSUE THREAD

**Issue by renarena** [BeardedTek-com/fEVR#97]
Time shown in events list is off several hours,

**Comment by BeardedTek** [comment]
Yes, it's hard coded to Alaska time at the moment.  I'll add an option to change timezones in the next release.  It's state fair time so it may be a few days.
